content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,40 @@ menu:
 weight: 101
 ---
 
+## v2.0.3 General Availability [2020-12-08]
+
+### Breaking Changes
+
+Previously, `influxd upgrade` would attempt to write upgraded `config.toml` files into the same directory as the source
+`influxdb.conf` file. If this failed, a warning would be logged and `config.toml` would write into the `HOME` directory.
+
+This release breaks this behavior in two ways:
+- By default, `config.toml` writes into the same directory as the Bolt DB and engine files (`~/.influxdbv2/`)
+- If writing upgraded config fails, the `upgrade` process exits with an error instead of falling back to the `HOME` directory
+
+Use the new `--v2-config-path` option to override the output path for upgraded config if they can't or don't
+want to use the default.
+
+### Features
+
+- Allow password to be specified as a CLI option in `influx v1 auth create`.
+- Allow password to be specified as a CLI option in `influx v1 auth set-password`.
+- Allow for users to specify where V2 config should be written in `influxd upgrade`.
+- Implement delete with predicate.
+- Improve ID-related error messages for `influx v1 dbrp` commands.
+
+### Bug Fixes
+
+- Use V2 directory for default V2 config path in `influxd upgrade`.
+- Don't log bodies of V1 write requests.
+- Fix panic when writing a point with 100 tags.
+- Ensure KV index walks only select exactly-matched keys.
```

Comment:
...index matches exact foreign keys only?

Reply:
If we want a more user-friendly description, we could mention the specific error that led Stuart to make this fix, something like:
```
Fix validation of existing DB names when creating DBRP mappings
```